openig-openam/src/test/java/org/forgerock/openig/openam/PolicyEnforcementFilterTest.java

```diff
@@ -425,7 +426,7 @@ public void shouldSucceedToUseCacheForRequestedResourceAllowed() throws Exceptio
                           resourceRequest,
                           next).get();
             // Then
-            verify(requestHandler).handleAction(any(PolicyDecisionContext.class), any(ActionRequest.class));
+            verify(requestHandler).handleAction(any(AttributesContext.class), any(ActionRequest.class));
```

Comment:
Could you please explain why you have changed `PolicyDecisionContext.class` to `AttributesContext.class`?